sql/core/src/main/scala/org/apache/spark/sql/catalyst/SQLBuilder.scala

```diff
@@ -193,6 +196,9 @@ class SQLBuilder(logicalPlan: LogicalPlan) extends Logging {
     case OneRowRelation =>
       ""
 
+    case Hint(_, _, child) =>
```

Comment:
Ur, it's added by previous advice comments.
No problem. I'll move that into `CheckAnalysis`.